src/Frontend/Core/Engine/Model.php

```diff
@@ -326,6 +329,29 @@ public static function getModuleSettings($module)
     }
 
     /**
+     * Creates router for module based on routing file
+     *
+     * @param $module
+     * @return null|Router
+     */
+    public static function getModuleRouter($module)
+    {
+        if (empty($module)) {
+            return null;
+        }
+
+        $file = FRONTEND_MODULES_PATH . '/' . $module . '/Resources/config/routing.yml';
+
+        $fs = new Filesystem();
+
+        if ($fs->exists($file)) {
+            return new Router(new YamlFileLoader(new FileLocator()), $file);
+        }
+
+        return null;
```

Comment:
This statement is not needed. If you don't add a return statement in a method, it will automatically return null.